src/inet.c

```diff
@@ -649,25 +665,18 @@ conn_t *pr_inet_create_conn_portrange(pool *p, const pr_netaddr_t *bind_addr,
         }
       }
 
-      c = init_conn(p, -1, bind_addr, ports[i], FALSE, FALSE);
-      if (c == NULL &&
-        /* Note that on Solaris, bind(2) might fail with EACCES if the
-         * randomly selected port for e.g. passive transfers is used by
-         * NFS.  Thus, for Solaris only, we treat EACCES as the same as
-         * EADDRINUSE.  Silly Solaris.
-         */
-#ifdef SOLARIS2
-          (inet_errno != EADDRINUSE && inet_errno != EACCES)) {
```

Comment:
This block of comment+code already happens in `init_conn()` anyway, so I feel it should be removed.  As should the abrupt "kill this session if there's any other error" behavior.  That's unnecessarily rude to clients.